clients/fides-js/src/lib/gpp/us-notices.ts

```diff
@@ -83,6 +83,10 @@ export const setGppNoticesProvidedFromExperience = ({
   const gppSection = FIDES_REGION_TO_GPP_SECTION[gppRegion];
 
   if (!gppSection) {
+    if (experience?.gpp_settings?.us_approach === GPPUSApproach.STATE) {
+      cmpApi.setApplicableSections([-1]);
```

Comment:
Do we even need this `if`?

In other words: should we just _always_ default it to -1 here if later in the code we end up initializing other sections?

Reply:
So this `if` allows us to exclude TCF. If we remove the `if`, then if both TCF and GPP is enabled, with a user visiting from the EU, this will line will set `-1`, overriding the `2` that was for TCF. 

For GPP with TCF, we do the following if we detect a TC string:

```
const tcSet = setTcString(event, cmpApi);
      if (tcSet) {
        cmpApi.setApplicableSections([TcfEuV2.ID]);
      }
```

So we wouldn't want to override that, if set.

Maybe instead of checking for `experience?.gpp_settings?.us_approach === GPPUSApproach.STATE`, I should check whether `tcSet` is false, then `cmpApi.setApplicableSections([-1])`. Does that make more sense to you?

Reply:
Oh ok, that comes down to order of initializations then - it looked like this might be running before anything else.

Your existing logic is OK then, it's specific and accurate.